packages/connectkit/src/index.ts

```diff
@@ -1,22 +1,28 @@
+export type * as Types from "./types";
+
+// Configure Daimo Pay
+export { DaimoPayProvider } from "./components/DaimoPay";
 export { default as getDefaultConfig } from "./defaultConfig";
-export * as Types from "./types";
-export { wallets } from "./wallets";
 
-// TODO: remove Context and usePayContext exports following SDK refactor.
-export {
-  Context,
-  DaimoPayProvider,
-  usePayContext,
-} from "./components/DaimoPay";
+// Pay button
 export { DaimoPayButton } from "./components/DaimoPayButton";
+
+// Hooks to track payment status + UI status.
+export { useDaimoPayStatus } from "./hooks/useDaimoPayStatus";
+
+/** TODO: replace with useDaimoPay() */
 export { useModal } from "./hooks/useModal";
 
+// These first two just return configured wagmi chains = not necessarily
+// supported by Daimo Pay.
```

Comment:
What do you mean? If dev uses `getDefaultConfig` as we intend them to those are equivalent?